index.html

```diff
@@ -10,45 +10,71 @@
 
   <body>
     <header class="header">
-      <h1>QUIZ-APP</h1>
+      <img src="images/piggy-header-no.png" alt="Logo der Quiz-App" />
     </header>
 
     <main>
       <section class="question-card">
-        <p class="question-number">Frage 1</p>
+        <img
+          class="bookmark-corner"
+          src="images/bookmark-icon.png"
+          alt="bookmark"
+        />
+
         <p class="question-text">lorem ipsum ?</p>
-        <button type="button">Show answer</button>
-        <p>#html</p>
-        <p>#flexbox</p>
-        <p>#css</p>
+        <button class="button" button type="button">Show answer</button>
+        <ul class="tags">
+          <li>#html</li>
+          <li>#flexbox</li>
+          <li>#css</li>
+        </ul>
       </section>
 
       <section class="question-card">
-        <p class="question-number">Frage 2</p>
+        <img
+          class="bookmark-corner"
+          src="images/bookmark-icon.png"
+          alt="bookmark"
+        />
+
         <p class="question-text">lorem ipsum bla bla bla ?</p>
-        <button type="button">Show answer</button>
-        <p>#html</p>
-        <p>#flexbox</p>
-        <p>#css</p>
+        <button class="button" button type="button">Show answer</button>
+        <ul class="tags">
+          <li>#html</li>
+          <li>#flexbox</li>
+          <li>#css</li>
+        </ul>
       </section>
 
       <section class="question-card">
-        <p class="question-number">Frage 3</p>
+        <img
+          class="bookmark-corner"
+          src="images/bookmark-icon.png"
+          alt="bookmark"
+        />
+
         <p class="question-text">lorem ipsum bla bla bla ?</p>
-        <button type="button">Show answer</button>
-        <p>#html</p>
-        <p>#flexbox</p>
-        <p>#css</p>
+        <button class="button" type="button">Show Answer</button>
+
+        <ul class="tags">
+          <li>#html</li>
+          <li>#flexbox</li>
+          <li>#css</li>
+        </ul>
       </section>
     </main>
 
     <footer>
       <nav>
-        <ul>
-          <li><a href="#">Home</a></li>
-          <li><a href="#">Bookmarks</a></li>
-          <li><a href="#">Profile</a></li>
-        </ul>
+        <a href="#"
+          ><img class="home" src="images/home-icon.png" alt="Home"
+        /></a>
+        <a href="#"
+          ><img class="bookmark" src="images/bookmark-icon.png" alt="Bookmarks"
+        /></a>
+        <a href="#"
+          ><img class="profile" src="images/profile-icon.png" alt="Profile"
+        /></a>
```

Comment:
you can only "click" on the image, but maybe it would be good to be able to click on the whole section?